### PR TITLE
PMM-8006 mysql with tls UI tests

### PR DESCRIPTION
**Pull request by IvanaBrecherova**

### CODE REVIEW

**Comment by IvanaBrecherova** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/9455/tests

File: tests/pages/remoteInstancesPage.js [hunk 1]
```diff
@@ -104,6 +108,9 @@ module.exports = {
     subscriptionID: '$azure_subscription_id-text-input',
     tableStatsGroupTableLimit: '$tablestats_group_table_limit-number-input',
     tenantID: '$azure_tenant_id-text-input',
+    tlscaInput: locate('$tls_ca-textarea-input'),
```
Comment (yurkovychv) [inline tests/pages/remoteInstancesPage.js:111]: there is no need to use `locate()`

File: tests/pages/remoteInstancesPage.js [hunk 2]
```diff
@@ -163,18 +171,39 @@ module.exports = {
...
   },
 
+  async fillTLS(file, field) {
+    const certificateData = await I.verifyCommand(`cat /tmp/ssl/pmm-ui-tests/testdata/mysql/ssl-cert-scripts/certs/${file}`);
+
+    I.waitForVisible(field, 30);
+    I.fillField(field, certificateData);
+  },
+
+  fillEnvironmentAndCluster(serviceName) {
```
Comment (yurkovychv) [inline tests/pages/remoteInstancesPage.js:181]: is there a need to have this logic just to have environment and cluster fields filled?

**Review by yurkovychv — CHANGES_REQUESTED** [review]
there is failed STT test in Jenkins, did you also check other jobs?

**Comment by IvanaBrecherova** [comment]
@yurkovychv Which jobs do you mean? :) the STT issue happen every second time, see this tests: https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/9452/tests same branch and no so big change. :) Who is STT master? :)

File: tests/pages/remoteInstancesPage.js (hunk 2 above)
Comment (IvanaBrecherova) [reply tests/pages/remoteInstancesPage.js:181]: Yes, I want to use all the steps to fill fields for MySQL but the only difference is the cluster and environment. And having a Switch statement inside another switch statement looks weird. :)

Comment (yurkovychv) [reply tests/pages/remoteInstancesPage.js:181]: As I recall those fields are not required. Do you use those fields to filter or what? If not, then I don't really see a reason to add more complexity here @puneet0191

Comment (IvanaBrecherova) [reply tests/pages/remoteInstancesPage.js:181]: I don't use them now. I will explain problem which can happen: The MySQL with TLS will be added, and MySQL without TLS not. Then we have a test to check QAN page for MySQL without TLS and we are checking that by cluster or environment. Then the test for check QAN will be false positive.

Comment (yurkovychv) [reply tests/pages/remoteInstancesPage.js:181]: Ok, but can you filter QAN by service name?

Comment (IvanaBrecherova) [reply tests/pages/remoteInstancesPage.js:181]: I can :) It refactoring of older tests.

Comment (puneet0191) [reply tests/pages/remoteInstancesPage.js]: env and cluster fields only needed to check if they are rightly available in QAN filter section, but if you want to check QAN queries you can use the service name also

**Comment by IvanaBrecherova** [comment]
Nightly job: https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-test-nightly/detail/pmm2-ui-test-nightly/621/tests/
Upgrade job: https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-upgrade-tests/detail/pmm2-upgrade-tests/4700/tests
OVF: https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/9467/pipeline

**Comment by IvanaBrecherova** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/9473/tests

File: tests/remoteInstances/remoteInstancesHelper.js [hunk 3]
```diff
@@ -239,6 +239,7 @@ module.exports = {
     postgresql: (remoteInstanceStatus.postgresql.pdpgsql_13_3.enabled ? 'postgresql_remote_new' : undefined),
     proxysql: (remoteInstanceStatus.proxysql.proxysql_2_1_1.enabled ? 'proxysql_remote_new' : undefined),
     postgresGC: (remoteInstanceStatus.gc.gc_postgresql.enabled ? 'postgresql_GC_remote_new' : undefined),
+    mysqlTLS: (remoteInstanceStatus.gc.gc_postgresql.enabled ? 'mysql_tls_remote_new' : undefined),
```
Comment (yurkovychv) [inline tests/remoteInstances/remoteInstancesHelper.js:242]: `gc_postgresql` - typo?

**Pull request closed without merging**